viewer/wasm/README.md

```diff
@@ -0,0 +1,19 @@
+# Reveal Internal Webassembly Module
+
+This crate contains all the embedded webassembly code for Reveal.
+
+## Build
+
+In order to build this crate, you will need to use Rust nightly. Using `rustup`, you can enable this by issuing
+
+```
+$ rustup override set nightly
+```
+
+in this directory.
+
+To build the crate for the web, we use `wasm-pack`:
+
+```
+$ wasm-pack build --target web
+```
```

Comment:
I guess you also need to install rust to build this? Some notes in the /viewer readme about this setup should probably be there.

Reply:
I've removed this README and added more information to the one in `/viewer` instead.